configs/config_sandbox/tf_version_debug/train-small-3class_tfversion_debug.yaml

```diff
@@ -0,0 +1,17 @@
+train_config:
+  model_id_prefix: 'segmentation-model-small-3class_tfversion_debug'
+  dataset_id: 'dataset-small-3class_tfversion_debug'
+  segmentation_model:
+    model_name: 'Unet'
+    model_parameters:
+      backbone_name: 'vgg16'
+      encoder_weights: Null   # Null (random initialization) or 'imagenet' (pre-training on ImageNet)
```

Comment:
note that imagenet requires RGB images for user. we'll have to standardize these configs, well update the standard config really after the near-term issues are done

Reply:
can I just remove the imagenet commented part?

Reply:
i was thinking about keeping it in case someone in the future has RGB datasets, so it wouldn't be forgotten. tho that's not our problem i guess, as the source can always be traced. the options also may change here once we allow additional architectures